package test1;

import static aritmetica.Aritmetica.division;

/**
 *
 * @author Viviana
 * 
 */
public class TestExcepciones {
 /* public static void main(String[] args) {
        int resultado = 0;
//        try{
           // resultado = Aritmetica.division(resultado,resultado);
           resultado = division(10,0);
//        }catch(Exception e){
//            System.out.println("Ocurrio un error: ");
//            e.printStackTrace(System.out);
//            System.out.println(e.getMessage());
//        }
        
        
        System.out.println("resultado = " + resultado);
    }
   */ 
}
